scripts/check_release_artifacts.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# should be run in CI or manually as:
+# ./check_release_artifacts.sh v2.3.1
```

Comment:
could you please remove the `v` from the parameter, it's implementation/naming detail.

Reply:
But that's what comes from CIRCLE_TAG, and could be used as here:
https://circleci.com/gh/aeternity/aeternity/22338
I'll post a pull request there later

Reply:
🤔 yeah, true. We could tag (and validate) v2.3.1-beta2 as well. 👍